.travis.yml

```diff
@@ -2,8 +2,13 @@ sudo: required
 
 language: go
 
-script: 
-# Run unit and integration tests
-- cd user && docker-compose up --build user && cd ..
-- cd auth && docker-compose up --build auth && cd ..
-- cd match && docker-compose up --build match && cd ..
+jobs:
+  include:
+  - stage: test
+    name: tests
+    # Run unit and integration tests
+    script: for svc in user auth match; do cd $svc && docker-compose up --build $svc && cd ..; done
```

Comment:
This can’t be a for loop - we need it to fail if any of the iterations fail and a loop only returns the exit code of the last iteration

Reply:
[Travis only allows one command per script block](https://docs.travis-ci.com/user/deployment/script/) (at least in stages), so I've separated them out into different build stages - also means they can be parallelised